File: Solution/Javascript/LCP 1. Guess Numbers.js
/**
 * @param {number[]} guess
 * @param {number[]} answer
 * @return {number}
 */
var game = function (guess, answer) {
    let res = 0;
    for (i = 0; i < guess.length; i++) {
        if (guess[i] == answer[i]) {
            res++
        }
    }
    return res;
};

File: Solution/Go/LCP 1. Guess Numbers.go
package main

import (
	"fmt"
)

func game(guess []int, answer []int) int {
	count := 0
	for k, v := range guess {
		if v == answer[k] {
			count++
		}
	}
	return count
}

func main() {
	guess := []int{1, 2, 3}
	answer := []int{1, 2, 3}
	res := game(guess, answer)
	fmt.Print(res)
}
